app/models/group.rb

```diff
@@ -911,9 +912,13 @@ def self.owner_of(groups, user)
     self.member_of(groups, user).where("gu.owner")
   end
 
+  def cache_group_users_for_destroyed_event
+    @cached_group_user_ids = group_users.pluck(:user_id)
```

Comment:
Without this I would need to check permissions of every user in chat if a group is destroyed, since all the dependent destroys remove any clues